python/mit/__init__.py

```diff
@@ -291,7 +290,7 @@ def __str__(self):
         return str(self.get())
 
     def get(self):
-        return self.getter(self.state)
+        return int(self.getter(self.state))
 
```

Comment:
The `int()` call is interesting. It feels like it might be possible to make `self.getter` return a Python int when called. One way to do this is by setting `self.getter.errcheck`, but we think that is not necessary for e.g. `c_int32`, so perhaps we should do whatever `c_int32` does?

Reply:
I think what `c_int32` does is have an entire suite of numeric methods.